app/assets/stylesheets/views/article.scss

```diff
@@ -254,6 +254,7 @@
 .reaction-drawer .crayons-reaction {
   border: 1px solid var(--card-bg);
   flex-direction: column;
+  padding: 8px 8px 4px 8px;
```

Comment:
This changes view for web too. 

### Result from this PR
<img width="451" alt="Screenshot 2023-02-09 at 2 36 49 PM" src="https://user-images.githubusercontent.com/9396084/217768784-0c1c5fd6-bde8-4b3d-8800-f5e601c3fbe4.png">


### Result from Forem.team
<img width="343" alt="Screenshot 2023-02-09 at 2 44 10 PM" src="https://user-images.githubusercontent.com/9396084/217768840-3afcb37c-1c16-4597-afdd-1c145388380a.png">